R/system.R

```diff
@@ -705,6 +705,50 @@ getDBConnection <- function(type, host = NULL, port = "", databaseName = "", use
         connection_pool[[key]] <- conn
       }
     }
+  } else if (type == "mssqlserver") {# The type sqlserver is already used for RODBC based one and "mssqlserver" is passed from Exploratory Desktop.
+
+    # if platform is Linux use predefined one
+    if(Sys.info()["sysname"]=="Linux"){
+      driver <-  "ODBC Driver 17 for SQL Server";
+    }
+    if(!requireNamespace("DBI")){stop("package DBI must be installed.")}
+    if(!requireNamespace("odbc")){stop("package RPostgres must be installed.")}
```

Comment:
Please add comment on why we are using odbc instead of RODBC here.

Reply:
Added comment.